tiatoolbox/dataloader/wsireader.py

```diff
@@ -135,6 +135,12 @@ def save_tiles(self, tile_format=".jpg", verbose=True):
             ...     tile_read_size_w=2000)
             >>> wsi_obj.save_tiles()
 
+        Examples:
+            >>> from tiatoolbox.dataloader import wsireader
+            >>> wsi_obj = wsireader.WSIReader(input_dir="./",
```

Comment:
The input is changed to input_path from JP2 branch. Once that's merged with develop probably you can update it.